.drone.yml

```diff
@@ -11,9 +11,9 @@ pipeline:
       event: push
       branch: master
     image: quay.io/uswitch/aws-cli:1.15.72
-    secrets: [ AWS_SECRET_ACCESS_KEY, AWS_ACCESS_KEY_ID ]
+    secrets: [AWS_SECRET_ACCESS_KEY, AWS_ACCESS_KEY_ID]
     commands:
-      - "aws s3 cp --region eu-west-1 --recursive ./static/icons s3://trustyle-icons --cache-control max-age=604800 --metadata '{\"x-amz-meta-surrogate-control\": \"max-age=604800\"}' --acl=public-read"
+      - 'aws s3 cp --region eu-west-1 --recursive ./static/icons s3://trustyle-icons --cache-control max-age=604800 --metadata ''{"x-amz-meta-surrogate-control": "max-age=604800"}'' --acl=public-read'
```

Comment:
Why did you change this line? I don't think the `.drone.yml` file needs changing

Reply:
i didnt change anything, will revert it

Reply:
also ive left types folder since i didn't know do you need it

Reply:
resolved